tablo/migrations/0006_alter_geom_col_type.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from tablo.models import GEOM_FIELD_NAME, WEB_MERCATOR_SRID
+
+forward_sql = """
+    DO
+    $$
+    DECLARE
+        table_name name;
+    BEGIN
+        FOR table_name IN
+            SELECT tablo_featureservicelayer.table FROM tablo_featureservicelayer
+        LOOP
+            EXECUTE format('ALTER TABLE %I ALTER COLUMN {geom_col} TYPE geometry(''GEOMETRY'', {srid});', table_name);
+        END LOOP;
+    END;
+    $$
+    LANGUAGE plpgsql;
+    """.format(geom_col=GEOM_FIELD_NAME, srid=WEB_MERCATOR_SRID)
+
+# FIXME In case of reverse, shall we drop non-point datasets and their relevant featureservice(layers)?
```

Comment:
I'd say (in this case), we either leave it like you have it or get rid of the reverse entirely, and have it be something you can't rollback.